### PR TITLE
Update common-windows-issues-key-management-server-activation.md

### PR DESCRIPTION
**Pull request by Riiskyy**
Hi I am a racker, Thomas Williams, working from my personal GitHub account. I have updated this article as some information was missing and some information was out of date. I have also forwarded an email to how-to@rackspace.com with 2 images that will need to be inserted where I have left comments.

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-8a6e99d6e1/how-to/). :bow:

File: content/cloud-servers/common-windows-issues-key-management-server-activation.md [hunk 1]
```diff
@@ -24,18 +24,9 @@ KMS.
    **Start > All Programs > Accessories**. Then, right-click **Command
    Prompt** and select **Run as Administrator**.
 
-2. Choose the data center in the following table that corresponds to
-   the location of your server, and run the applicable command from the
-   command prompt:
+2. Confirm that you can ping the Rackspace KMS server
```
Comment (catlook) [inline content/cloud-servers/common-windows-issues-key-management-server-activation.md]: ... Rackspace KMS server by running the following command:

File: content/cloud-servers/common-windows-issues-key-management-server-activation.md [hunk 2]
```diff
@@ -45,25 +36,57 @@ KMS.
...
-   | SYD (Sydney)    | `slmgr.vbs /skms winactivate.syd2.servers.rackspacecloud.com:1688` |
+        slmgr.vbs /skms kms.rackspace.com:1688
 
 4. Request activation from the KMS:
 
-       slmgr.vbs /ato
+        slmgr.vbs /ato
+
+    If you recieve the error  ``0xC004F074 The Key
+    Management Server (KMS) is unavailable`` skip to step 9
```
Comment (catlook) [inline content/cloud-servers/common-windows-issues-key-management-server-activation.md]: ... unavailable``, skip to step 9.

File: content/cloud-servers/common-windows-issues-key-management-server-activation.md [hunk 3]
```diff
@@ -45,25 +36,57 @@ KMS.
...
+
+5. If the device does not activate then the server may be set to MAK activation instead of KMS activation.
+    To confirm which activation method is set on the device run the following command:
```
Comment (catlook) [inline content/cloud-servers/common-windows-issues-key-management-server-activation.md]: ... device, run the ...

File: content/cloud-servers/common-windows-issues-key-management-server-activation.md [hunk 4]
```diff
@@ -45,25 +36,57 @@ KMS.
...
+
+        SLMGR -dlv
+
+    Look for the 'Product Key Channel' setting. 'Volume:GVLK' means the device is set to KMS activation,
```
Comment (catlook) [inline content/cloud-servers/common-windows-issues-key-management-server-activation.md]: ... the **Product Key Channel** setting. The value **Volume:GVLK** means that the device is set to **KMS activation**, ...

File: content/cloud-servers/common-windows-issues-key-management-server-activation.md [hunk 5]
```diff
@@ -45,25 +36,57 @@ KMS.
...
+    'Volume:MAK' means the device is set to MAK activation.
```
Comment (catlook) [inline content/cloud-servers/common-windows-issues-key-management-server-activation.md]: and the value **Volume:MAK** means the device is set to **MAK activation**.

File: content/cloud-servers/common-windows-issues-key-management-server-activation.md [hunk 6]
```diff
@@ -45,25 +36,57 @@ KMS.
...
+
```
Comment (catlook) [inline content/cloud-servers/common-windows-issues-key-management-server-activation.md]: [Add line] The following images show sample outputs:

File: content/cloud-servers/common-windows-issues-key-management-server-activation.md [hunk 7]
```diff
@@ -45,25 +36,57 @@ KMS.
...
+    **KMS Activation Output**:
+    <!--insert KMS image here-->
+
+    **MAK Activation Output**:
+    <!--insert MAK image here-->
+
+6. If your device is set to MAK activation then you will need to set the device back to KMS activation.
```
Comment (catlook) [inline content/cloud-servers/common-windows-issues-key-management-server-activation.md]: ... set to **MAK activation**, you should set the device back to **KMS activation**. ...

File: content/cloud-servers/common-windows-issues-key-management-server-activation.md [hunk 8]
```diff
@@ -45,25 +36,57 @@ KMS.
...
+    First find and take note of the appropriate KMS client setup key from Microsoft: [KMS Client Setup Keys](https://technet.microsoft.com/library/jj612867.aspx)
+
+    To find wich Server edition you are running, run the following command and look for the section labelled 'OS name':
```
Comment (catlook) [inline content/cloud-servers/common-windows-issues-key-management-server-activation.md]: ... labelled **OS name**: ...

File: content/cloud-servers/common-windows-issues-key-management-server-activation.md [hunk 9]
```diff
@@ -45,25 +36,57 @@ KMS.
...
+
+        systeminfo | findstr OS
+
+    Example:
+
+        PS C:\Users\Administrator> systeminfo | findstr OS
+        OS Name:                   Microsoft Windows Server 2012 R2 Datacenter
+
+7. Set the device to KMS acivation using the key found in the above article with the following command:
```
Comment (catlook) [inline content/cloud-servers/common-windows-issues-key-management-server-activation.md]: ... to **KMS activation** by using the key found in the previously referenced article and entering the following ...

File: content/cloud-servers/common-windows-issues-key-management-server-activation.md [hunk 10]
```diff
@@ -45,25 +36,57 @@ KMS.
...
+
+        slmgr /ipk %key%
+
+    Make sure to replace **%key%** with the key from the Microsoft document.
+
+8. To activate the device run the below command:
```
Comment (catlook) [inline content/cloud-servers/common-windows-issues-key-management-server-activation.md]: ... device, run the following command: ...

File: content/cloud-servers/common-windows-issues-key-management-server-activation.md [hunk 11]
```diff
@@ -45,25 +36,57 @@ KMS.
...
 
-5. If step 4 returns the error ``0xC004F074 The Key
+9. If step 4 returned the error ``0xC004F074 The Key
    Management Server (KMS) is unavailable``, run the following command:
 
         w32tm /resync
 
-6. If the time on the cloud server is drastically different than
+10. If the time on the cloud server is drastically different than
      what is on the KMS the re-sync will fail.  At this point, you should
```
Comment (catlook) [inline content/cloud-servers/common-windows-issues-key-management-server-activation.md]: ... on the KMS, the re-sync ...

**Comment by Riiskyy** [comment]
Suggested changes have been committed.

**Comment by catlook** [comment]
merging as is, then I'll open new PR so I can add the images.

**Pull request merged**